src/tests/010-userTest.js

```diff
@@ -5,18 +5,12 @@ import mockData from './mock/mockData';
 
 chai.use(chaiHttp);
 chai.should();
-
 let token;
-
-
 const { expect } = chai;
-
 const {
-  usersSignin, unknownUserUpdate, invalidManager, userInvalidImage
+  unknownUserUpdate, invalidManager, userInvalidImage
```

Comment:
In order to avoid complexity, I suggest you don't destruct here, remember the mockdata is wrapped into an object, use it to call property of object you want.

Reply:
@NiyoEric Please explain on what you mean by complexity?
This is because I was trying to follow the ES6 principle of destructuring.

Reply:
@NiyoEric Thanks I now understand why